### PR TITLE
Greetings page

### PR DESCRIPTION
**Pull request by VictoriaKGN**
- designed and styled greetings page
- created Login and Signup paths so buttons work
- added font in global theme

image

### CODE REVIEW

File: client/src/pages/Greetings/Greetings.tsx [hunk 1]
```diff
@@ -1,16 +1,66 @@
...
  * greetings page component
  * @returns {JSX.Element} - greetings page
  */
 const Greetings = () => {
   return (
-    <StyledDiv>
-      <Header isLoggedIn={false} setIsLoggedIn={() => null} />
-    </StyledDiv>
+    <GreetingsDiv>
+      <LeftStack spacing={'3vh'} alignItems="center">
```
Comment (dlarocque) [inline client/src/pages/Greetings/Greetings.tsx:23]: Is it necessary to rename all the material components when we add some styling to them? For me it's a bit hard to tell what's what (e.g. BodyText is just styled Typography, LeftStack is a Stack, WelcomeHeader is also a Typography)

**Review by dlarocque — APPROVED** [review]

File: client/src/pages/Greetings/Greetings.tsx (hunk 1 above)
Comment (BarbzCodez) [reply client/src/pages/Greetings/Greetings.tsx:23]: Its not necessary but its easier to understand when you are reading just the component, also easier to reuse and be consistant, I kinda agreed with victoria if the div styling/option is more than 3 lines then then it should be a style

**Review by BarbzCodez — APPROVED** [review]
Makes sense :)

File: client/src/pages/Greetings/Greetings.tsx (hunk 1 above)
Comment (dlarocque) [reply client/src/pages/Greetings/Greetings.tsx:23]: Yeah agreed, the styles should be in a separate file, I just personally find the readability better when it's clear what MUI components are being used without having to see the definition.

If you guys both prefer it then LGTM

**Pull request merged**